go.mod

```diff
@@ -55,3 +55,5 @@ require (
 	k8s.io/client-go v0.21.3
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/flyteorg/flyteidl => github.com/flyteorg/flyteidl v0.21.20-0.20220111070000-bdd241a81330
```

Comment:
doesn't this need to be updated?

Reply:
Awaiting on the IDL merge to send an updated PR for this